mappings/net/minecraft/client/particle/Particle.mapping

```diff
@@ -0,0 +1,81 @@
+CLASS none/bnl net/minecraft/client/particle/Particle
+	FIELD A colorRed F
+	FIELD B colorGreen F
+	FIELD C colorBlue F
+	FIELD D colorAlpha F
+	FIELD E sprite Lnone/bxe;
+	FIELD H lerpX D
+	FIELD I lerpY D
+	FIELD J lerpZ D
+	FIELD K billboardNormal Lnone/bcv;
+	FIELD a EMPTY_BOUNDING_BOX Lnone/bcs;
+	FIELD b boundingBox Lnone/bcs;
+	FIELD c world Lnone/aiw;
+	FIELD d prevPosX D
+	FIELD e prevPosY D
+	FIELD f prevPosZ D
+	FIELD g posX D
+	FIELD h posY D
+	FIELD i posZ D
+	FIELD j velocityX D
+	FIELD k velocityY D
+	FIELD l velocityZ D
+	FIELD m onGround Z
+	FIELD n collidesWithWorld Z
+	FIELD o dead Z
+	FIELD p spacingXZ F
+	FIELD q spacingY F
+	FIELD r random Ljava/util/Random;
+	FIELD s tileU I
+	FIELD t tileV I
+	FIELD w age I
+	FIELD x maxAge I
+	FIELD z gravityStrength F
+	METHOD <init> (Lnone/aiw;DDD)V
+		ARG 0 world
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+	METHOD <init> (Lnone/aiw;DDDDDD)V
+		ARG 0 world
+		ARG 1 x
+		ARG 2 y
+		ARG 3 z
+		ARG 4 velocityX
+		ARG 5 velocityY
+		ARG 6 velocityZ
+	METHOD a tick ()V
```

Comment:
Also, `tick` is also less vague. If you insist though I can move it back to update.